src/main/resources/logback.xml

```diff
@@ -0,0 +1,26 @@
+<configuration>
```

Comment:
Implement the log rolling strategy by date

Reply:
fix